### PR TITLE
refactor: structure cleanup & capabilities -> packages

### PR DESCRIPTION
**Pull request by jeff-mccoy**
## Description

- rename `capabilities` -> `packages`
- move `./capabilities/` -> `./src/`
- refer to individual packages (Istio, Neuvector, etc) as source packages internally
- remove per-package changelogs, docs/adrs, boilerplate README contents
- remove stubbed folder paths in unfinished source packages
- various readme updates

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Other (security config, docs update, etc)

## Checklist before merging

- [x] Test, docs, adr added or updated as needed
- [x] [Contributor Guide Steps](https://github.com/defenseunicorns/uds-template-capability/blob/main/CONTRIBUTING.md)(https://github.com/defenseunicorns/uds-template-capability/blob/main/CONTRIBUTING.md#submitting-a-pull-request) followed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,25 +1,25 @@
...
+- [KeyCloak](https://www.keycloak.org/) - Identity & Access Management
+- [Kiali](https://kiali.io/) - Service Mesh Observability
+- [Kyverno](https://kyverno.io/) - Policy Engine
+- [Loki](https://grafana.com/oss/loki/) - Log Aggregation
+- [Metrics Server](https://github.com/kubernetes-sigs/metrics-server) - Metrics
+- [Neuvector](https://open-docs.neuvector.com/) - Container Security
+- [Prometheus Stack](https://github.com/prometheus-operator/kube-prometheus) - Monitoring
+- [Promtail](https://grafana.com/docs/loki/latest/send-data/promtail/) - Log Aggregation
+- [Tempo](https://grafana.com/docs/tempo/latest/getting-started/) - Tracing
+- [Velero](https://velero.io/) - Backup & Restore
```
Comment (mjnagel) [inline README.md:22]: Is there still value in capturing status here (the checkboxes)?

File: bundles/k3d-istio/uds-bundle.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 kind: UDSBundle
 metadata:
   name: k3d-core-istio
-  description: An example UDS bundle for deploying Istio from UDS Core on a development cluster
+  description: An UDS bundle for deploying Istio from UDS Core on a development cluster
```
Comment (mjnagel) [inline bundles/k3d-istio/uds-bundle.yaml]:
```suggestion
  description: A UDS bundle for deploying Istio from UDS Core on a development cluster
```

nit

**Review by zachariahmiller — DISMISSED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (jeff-mccoy) [reply README.md:22]: maybe? idk

File: bundles/k3d-istio/uds-bundle.yaml (hunk 2 above)
Comment (zachariahmiller) [reply bundles/k3d-istio/uds-bundle.yaml]: image

**Review by mjnagel — APPROVED** [review]
:shipit:

File: README.md (hunk 1 above)
Comment (mjnagel) [reply README.md:22]: Resolving - checkboxes would be pretty shortlived likely since we are going to knock the others out soon hopefully.

**Review by zachariahmiller — APPROVED** [review]
🚀

**Pull request merged**